usaspending_api/common/helpers/endpoint_documentation.py

```diff
@@ -90,7 +90,9 @@ def validate_docs(url, url_object, master_endpoint_list):
             absolute_endpoint_doc = os.path.join(BASE_DIR, endpoint_doc)
             if not case_sensitive_file_exists(absolute_endpoint_doc):
                 messages.append(
-                    "{}.endpoint_doc ({}) references a file that does not exist".format(qualified_name, url)
+                    "{}.endpoint_doc ({}) references a file that does not exist ({})".format(
+                        qualified_name, url, endpoint_doc
```

Comment:
I trust that having 3 variables and 2 `{}` works in format() for some reason I don't understand, but was this tested? (I can do it if it wasn't but it will take a while)

Reply:
> I trust that having 3 variables and 2 `{}` works in format() for some reason I don't understand, but was this tested? (I can do it if it wasn't but it will take a while)

I think you need more fingers.

Reply:
...what?

Reply:
```
"{1}.endpoint_doc ({2}) references a file that does not exist ({3})"
  1:qualified_name, 2:url,                                      3:endpoint_doc
```
The counts match?